CMakeModules/scripts/copydlls.cmake

```diff
@@ -0,0 +1,24 @@
+# this module expects
```

Comment:
Isn't dll  windows only concept ? what do we use it for ?

Reply:
Oh, I think this is superfluous, just accidentally captured from aleth along with secp256k1 =)